README.md

```diff
@@ -34,13 +34,15 @@ present 0.1.1
 Interpolate the standard output of arbitrary shell scripts into your markdown files
```

Comment:
I think we should add a library section in the readme as well, with a small example

Reply:
Agreed!